src/steps/success.ts

```diff
@@ -1,11 +1,19 @@
-import type { Context } from 'semantic-release';
+import type { BranchObject, Context } from 'semantic-release';
 import { execSync } from 'node:child_process';
 
 import prepareTags from './prepareTags';
 import type { PluginConfigType } from './types';
 
+function isPrerelease(branchObject: BranchObject) {
+  return branchObject.prerelease !== undefined && branchObject.prerelease !== false
```

Comment:
Could it be just like that? 🤔   
```suggestion
  return !!branchObject.prerelease
```
It's either a string, boolean or undefined therefore that should cover all cases.  
`undefined` -> `false`
`true` -> `true`
`''` (empty string) -> `false`
`'any-string'` -> `true`

What do you think? :D

Reply:
The `!!` approach has different behavior when `prerelease` is `''` (empty string). The explicit style is inspired by eslint's [strict-boolean-expressions](https://github.com/typescript-eslint/typescript-eslint/blob/766ab1de882b5af6d570988dd4ba7c30fb0ceabb/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md) rule, which I find more clearly communicates intent. 

That said, I'll switch to `!!` style as I'm not maintaining this codebase.

Reply:
Good point, question, should it be counted as a `prerelease` also if the `''` is an empty string? 🤔   
And no worries, just thinking about the edge cases, good job btw 💪